src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/PrisonService.kt

```diff
@@ -67,8 +68,12 @@ class PrisonService(
   }
 
   fun getSentencesAndOffences(bookingId: Long, filterActive: Boolean = true): List<SentenceAndOffences> {
-    return prisonApiClient.getSentencesAndOffences(bookingId)
+    val sentencesAndOffences = prisonApiClient.getSentencesAndOffences(bookingId)
       .filter { !filterActive || it.sentenceStatus == "A" }
+
+    offenceSdsPlusLookupService.populateSdsPlusMarkerForOffences(sentencesAndOffences)
```

Comment:
Can this blow up? I see getSentencesAndOffences is used in a number of places.

Reply:
Now throws a sensible error if it does blow up which will get propagated, it'll appear as any normal error does though to the user as does everything for the time being.